### ISSUE TITLE
[Screen Reader-Home Page-Blog]: Link Purpose for the all 'Read More' links is not descriptive to the users on the Blog page

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2647]
**User Experience:**
User who depends on Screen reader will feel difficulty to access the links if the purpose for the 'Read More' links are not descriptive.

**Test Environment:**
OS: Windows 10 RS2
Build: 15063.2
Browser: Edge
Tool Used: IE11 Developer Tool

**Pre-Requisite**: Turn on the Narrator

**Steps to reproduce:**
1. Navigate to url:- https://makecode.com/
2. Navigate to the Blog showing on header section and select it.
3-Verify, the link purpose is descriptive for the 'Read More' links or not.

**Actual Behavior:**
Link Purpose for the all 'Read More' links is not descriptive on the Blog page

**Expected Behavior:**
Link Purpose for the all 'Read More' links should be well descriptive so that user may get complete information about the links on the Blog page.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}

**Suggested Fix:**
[‎8/‎3/‎2017 1:10 PM]  Swati Sharma (HCL America Inc):  
1 -Providing link text that describes the purpose of a link for anchor elements (HTML)  <a href="routes.html"> 
   Current routes at Boulders Climbing Gym </a> Example 1: Clarifying the purpose of a link Example Code: <a href="http://example.com/WORLD/africa/kenya.elephants.ap/index.html"  
title="Read more about failed elephant evacuation"> 
 Evacuation Crumbles Under Jumbo load 
</a> 
2-Providing text alternatives for the area elements of image maps 
3-Using scripts to change the link text (choose short or long link text ) 
4- Use 'aria-labelledby' or 'aria-label' for link purpose
Key References for Fix:  
http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html  
 
**Please refer the Attachment:**
![mas2 4 4_homepage_blog](https://user-images.githubusercontent.com/30434417/28908914-c82ea3de-7843-11e7-9335-409bb7701c1d.jpg)

[MAS2.4.4_HomePage_Blog.zip](https://github.com/Microsoft/pxt/files/1196207/MAS2.4.4_HomePage_Blog.zip)

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is not fixed now. Hence reactivating the bug

**Comment by sonimishra** [comment]
Bug is verified on URL: https://makecode.microbit.org/acc and issue is fixed as now Link Purpose for the all 'Read More' links is descriptive to the users on the Blog page.I don't have permissions to add/Remove labels.

**Comment by samelhusseini** [comment]
Thanks for verifying, Closing as fixed.
